fe/fe-core/src/main/java/com/starrocks/connector/delta/DatabricksUnityMetastore.java

```diff
@@ -0,0 +1,158 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.connector.delta;
+
+import com.databricks.sdk.WorkspaceClient;
+import com.databricks.sdk.service.catalog.SchemaInfo;
+import com.databricks.sdk.service.catalog.TableInfo;
+import com.databricks.sdk.service.catalog.TableType;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Streams;
+import com.starrocks.catalog.Database;
+import com.starrocks.catalog.DeltaLakeTable;
+import com.starrocks.connector.ConnectorTableId;
+import com.starrocks.connector.HdfsEnvironment;
+import com.starrocks.connector.exception.StarRocksConnectorException;
+import io.delta.kernel.Scan;
+import io.delta.kernel.ScanBuilder;
+import io.delta.kernel.client.TableClient;
+import io.delta.kernel.data.FilteredColumnarBatch;
+import io.delta.kernel.data.Row;
+import io.delta.kernel.internal.InternalScanFileUtils;
+import io.delta.kernel.utils.CloseableIterator;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static com.starrocks.connector.PartitionUtil.toHivePartitionName;
+
+public class DatabricksUnityMetastore {
+    private static final Logger LOG = LogManager.getLogger(DatabricksUnityMetastore.class);
+    public static final String DATABRICKS_HOST = "databricks.host";
+    public static final String DATABRICKS_TOKEN = "databricks.token";
+    public static final String DATABRICKS_CATALOG_NAME = "databricks.catalog.name";
+    public static final String DATABRICKS_UNITY_METASTORE = "unity";
+    private final String catalogName;
+    private final String databricksCatalogName;
+
+    private final WorkspaceClient workspaceClient;
+    private final HdfsEnvironment hdfsEnvironment;
+
+    public DatabricksUnityMetastore(String catalogName, String databricksCatalogName,
+                                    WorkspaceClient workspaceClient,
+                                    HdfsEnvironment hdfsEnvironment) {
+        this.catalogName = catalogName;
+        this.databricksCatalogName = databricksCatalogName;
+        this.workspaceClient = workspaceClient;
+        this.hdfsEnvironment = hdfsEnvironment;
+    }
+
+    public List<String> getAllDatabaseNames() {
+        List<String> dbNames = Lists.newArrayList();
+        try {
+            dbNames = Streams.stream(workspaceClient.schemas().list(databricksCatalogName).iterator()).
+                    map(SchemaInfo::getName).collect(Collectors.toList());
+        } catch (Exception e) {
+            LOG.warn(e.getMessage());
```

Comment:
You should print more information not just print error messages.
And I have a question, when some exceptions happen, how does the client know the issues?